packages/storybook/src/schematics/storybook-configure/files/.storybook/config.js

```diff
@@ -0,0 +1,11 @@
+import { configure, addDecorator } from '@storybook/angular';
+import { withKnobs } from '@storybook/addon-knobs';
+
+function loadStories() {
+  const req = require.context('../src/lib', true, /\.stories\.ts$/);
```

Comment:
Is the path correct? Shouldn't it be '../libs'? There is no path like '../src/lib' in nx workspace.